requirements/common.in

```diff
@@ -23,6 +23,11 @@ jsx-lexer
 # Needed for manage.py
 ipython
 
+# Needed for compatibility with ipython < 7.20, which we cannot
+# install on Python <3.7; see
+# https://github.com/ipython/ipython/issues/12740
+jedi<0.18.0
```

Comment:
We could write `jedi<0.18.0;python_version<"3.7"` as a reminder that this can be removed when we upgrade Python.

Reply:
Installing it this way means that it doesn't get installed under Python 3.8, which doesn't end well.  I think the comment will need to suffice.

Reply:
I see. That's the kind of pip-compile bug I guess I should expect.